examples/exclude-words/create_wordlist.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+import glob
+import json
+import pathlib
+from pdfminer.high_level import extract_text
+
+
+def main(args):
+    file_path = args.file_path
+    frequency = int(args.minimal_frequency)
+    minimal_length = int(args.minimal_length)
```

Comment:
minimal or minimum? Also is there a reason why `frequency` didn't get to keep the `minimal` prefix but length did?

```suggestion
    frequency = int(args.minimal_frequency)
    minimal_length = int(args.minimal_length)
```